feature/chatting/src/commonMain/kotlin/com/nexters/emotia/feature/chatting/ChattingScreen.kt

```diff
@@ -23,23 +23,36 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.dp
-import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import com.nexters.emotia.core.designsystem.component.ChatBubble
 import com.nexters.emotia.core.designsystem.component.TypingIndicator
 import com.nexters.emotia.core.designsystem.theme.LocalEmotiaColors
+import com.nexters.emotia.feature.chatting.contract.ChattingIntent
+import com.nexters.emotia.feature.chatting.contract.ChattingSideEffect
 import org.koin.compose.viewmodel.koinViewModel
+import org.orbitmvi.orbit.compose.collectAsState
+
 
 @Composable
 fun ChattingScreen(
     onNavigateToResult: () -> Unit,
     modifier: Modifier = Modifier,
     viewModel: ChattingViewModel = koinViewModel()
 ) {
-    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+    val uiState by viewModel.collectAsState()
     val colors = LocalEmotiaColors.current
     val lazyListState = rememberLazyListState()
 
-    // 새 채팅 올 때마다 스크롤
+    // SideEffect 처리
+    LaunchedEffect(Unit) {
+        viewModel.container.sideEffectFlow.collect { sideEffect ->
+            when (sideEffect) {
+                is ChattingSideEffect.ShowError -> {
+                    // TODO : 에러 디자인시스템 요청
+                }
+            }
+        }
+    }
```

Comment:
Orbit에는 collectSideEffect 라는 문법이 있으니 이걸 사용해보세요

```suggestion
        viewModel.collectSideEffect { sideEffect ->
            when (sideEffect) {
                is ChattingSideEffect.ShowError -> {
                    // TODO : 에러 디자인시스템 요청
                }
            }
        }
    }
```

Reply:
refCountSideEffectFlow 라는게 내장되어있네요, repeatOnLifecycle하면서 일회성 이벤트 조합이네 신세계다 반영해둘게요 ~ 👍🏻